### DISCUSSION TITLE
Mxnet official website before the search function area ran where? Updated?

### DISCUSSION THREAD

**Discussion by mifan0208** [apache/incubator-mxnet#16237] [Q&A] [answered]

[a comment by an automated account is not shown]

**Comment by lanking520** [comment]
Hi @mifan0208 Could you provide more information for your question?

**Comment by wkcn** [comment]
mifan0208’s question may be ‘where is the search box on the main page of MXNet website’.

The new search box only exists in the documents pages, like http://mxnet.incubator.apache.org/api/python/docs/api/

Some users report that they could not find the search box in the new website, and the old search box is more convenient than the new one. I also hope that the search box exists on the right-top of the main page of the new website.

**Comment by szha** [comment, marked as answer]
Global search feature is now available on MXNet website.
